packages/esm-service-queues-app/src/active-visits/active-visits-table.component.tsx

```diff
@@ -265,6 +265,13 @@ function ActiveVisitsTable() {
     launchAddProviderRoomModal,
   ]);
 
+  useEffect(() => {
+    // Clean up the modal state when navigating away or resetting provider room
+    return () => {
+      setProviderRoomModalShown(false);
+    };
+  }, [setProviderRoomModalShown]);
```

Comment:
I'm not too familiar with the code. @makombe when should the modal be shown? This useEffect callback will only be run when navigating away from the page that has the active-visits-table. This will not prevent multiple modals from showing, right?